keyboards/nightly_boards/n2/readme.md

```diff
@@ -0,0 +1,18 @@
+# Alter
+
+![n2](https://i.imgur.com/iHOxmHi.png)
```

Comment:
Swaps this image for a smaller version.

```suggestion
![n2](https://i.imgur.com/iHOxmHil.png)
```

Reply:
https://github.com/qmk/qmk_firmware/pull/9862/commits/b53013853d923d898dee317a66ed8c05eea79334